roles/cinder-common/tasks/main.yml

```diff
@@ -41,6 +41,10 @@
 - include: ceph_integration.yml
   when: ceph.enabled
 
+- name: Error out when Swift HAProxy VIP is not defined
+  fail: msg="Swift HAProxy VIP must be set for Cinder backup to function correctly"
+  when: swift.enabled and not endpoints.swift.haproxy_vip
```

Comment:
haproxy_vip is defined in endpoints role. however it is not set since it varies for each environment, so check is to verify haproxy_vip is not empty.. I will make change to add bool filter